docs/tutorial/quick-start.rst

```diff
@@ -28,14 +28,13 @@ For more in-depth installation instructions, refer to :ref:`rocm-install-overvie
                     .. code-block:: bash
                         :substitutions:
 
+                        sudo apt update
                         sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
-                        sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
+                        # Add the current user to the render and video groups (see prerequisites).
+                        sudo usermod -a -G render,video $LOGNAME
                         wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
                         sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb
-                        sudo apt update
```

Comment:
The "sudo apt update" line needs to stay.  Do not remove it.

Reply:
@dbieleck We're making `sudo apt update` the first step. Is that fine with you?

```shell
sudo apt update
sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
# Add the current user to the render and video groups (see prerequisites).
sudo usermod -a -G render,video $LOGNAME
wget https://repo.radeon.com/amdgpu-install/6.1.2/ubuntu/jammy/amdgpu-install_6.1.60102-1_all.deb
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt install amdgpu-dkms rocm
```

Reply:
No.  You need to have the update after the install of the amdgpu-install script, or the instructions will fail:

...
sudo apt install ./amdgpu-install_6.1.60102-1_all.deb
sudo apt update
sudo apt install amdgpu-dkms rocm

The apt update on the top line can remain.

Reply:
@dbieleck Ohh ok, thanks